### PR TITLE
Remove TELEMETRYEVENTSOURCE_PUBLIC flag to resolve namespace conflict with external TelemetryEventSource.cs

### PR DESCRIPTION
**Pull request by ryfu-msft**
The TelemetryEventSource from the Msix-packaging library causes a namespace conflict with other projects that want to implement their own managed-code telemetry. By removing this flag and making this class internal, it allows us to create our own telemetry implementation without a namespace conflict.

### CODE REVIEW

**Comment by palenshus** [comment]
This class gets replaced at release build, so this wouldn't have an effect, there's also other things that are public in this class that could be internal too. Better to remove the flag set here: https://github.com/microsoft/msix-packaging/blob/master/tools/utils/Utils/Utils.csproj#L27

**Comment by ryfu-msft** [comment]
> This class gets replaced at release build, so this wouldn't have an effect, there's also other things that are public in this class that could be internal too. Better to remove the flag set here: https://github.com/microsoft/msix-packaging/blob/master/tools/utils/Utils/Utils.csproj#L27

Reverted changes, and removed defined flag (TELEMETRYEVENTSOURCE_PUBLIC)

File: tools/utils/Utils/Utils.csproj [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
     <!--TELEMETRYEVENTSOURCE_PUBLIC is used internally in TelemetryEventSource to make the class public-->
-    <DefineConstants>TRACE;DEBUG;TELEMETRYEVENTSOURCE_USE_NUGET;TELEMETRYEVENTSOURCE_PUBLIC</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;TELEMETRYEVENTSOURCE_USE_NUGET</DefineConstants>
```
Comment (lechacon) [inline tools/utils/Utils/Utils.csproj:27]: Remove all uses of this. There's another one further down.

Comment (lechacon) [reply tools/utils/Utils/Utils.csproj:27]: And update the comment above too, please.

**Review by lechacon — APPROVED** [review]

File: tools/utils/Utils/Utils.csproj (hunk 1 above)
Comment (ryfu-msft) [reply tools/utils/Utils/Utils.csproj:26]: removed all TelemetryEventSource_Public flag uses and the unnecessary comment

**Pull request merged**